Fabulous.XamarinForms/templates/content/blank/NewApp.Android/NewApp.Android.fsproj

```diff
@@ -68,47 +83,11 @@
     <AndroidResource Include="Resources\values\colors.xml" />
     <None Include="packages.config" />
   </ItemGroup>
-  <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.FSharp.targets" />
```

Comment:
This one is required. We should keep it.

Reply:
Readded